website/docusaurus.config.ts

```diff
@@ -69,7 +69,8 @@ const config: Config = {
     footer: {
       style: 'dark',
       links: [],
-      copyright: `Copyright © ${new Date().getFullYear()} Azure Container Upstream.`,
+      copyright: `Copyright Dalec a Series of LF Projects, LLC
```

Comment:
```suggestion
      copyright: `Copyright © ${new Date().getFullYear()} Dalec, a Series of LF Projects, LLC.
For website terms of use, trademark policy and other project policies please see lfprojects.org/policies/.`
```

I recommend keeping the dynamic date logic.

Reply:
Yeah it looks like it. So I guess the year is irrelevant now? I'll close the other PR now.